.bit/.progress

```diff
@@ -0,0 +1 @@
+0
```

Comment:
I think you left the bot installed on your account @shreythecray, so it's opening things as if you started a course. You'll need to delete .progress